PrismSampleApp/PrismSampleApp/ViewModels/ViewModelBase.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Prism.Commands;
+using Prism.Mvvm;
+using Prism.Navigation;
+using PrismSampleApp.Model;
+using PrismSampleApp.Services.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace PrismSampleApp.ViewModels
+{
+    public class ViewModelBase : BindableBase, IInitialize, INavigationAware, IDestructible,IWebApiService
```

Comment:
Don't inherit service to ViewModel base. Why are we doing this?

Reply:
Service has been removed.